eva-accession-release-automation/run_release_in_embassy/copy_accessioning_collections_to_embassy.py

```diff
@@ -119,6 +127,8 @@ def copy_accessioning_collections_to_embassy(private_config_xml_file, profile, t
             mongo_data_copy_to_remote_host(mongo_port, private_config_xml_file, profile, assembly_accession,
                                            taxonomy_id, collections_to_copy_map, dump_dir, destination_db_name)
             exit_code = 0
+            if remote_db_is_empty(mongo_port, assembly_accession, destination_db_name):
+                raise Exception
```

Comment:
Maybe include a message about what went wrong

Reply:
I would but it is getting caught and re-raised right away overwriting the message.